cfn-resources/integration-tests/project/project_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package project
+
+import (
+	ctx "context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	cfn "github.com/aws/aws-sdk-go-v2/service/cloudformation"
+	"github.com/mongodb/mongodbatlas-cloudformation-resources/integration-tests/util"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type TestContext struct {
+	cfnClient   *cfn.Client
+	atlasClient *mongodbatlas.Client
+	template    []byte
+	stackName   string
+	projectName string
+	projectId   string
+	err         error
+}
+
+func TestProjectCFN(t *testing.T) {
+	ctx := new(TestContext)
+	ctx.setUp()
+	testIsTemplateValid(t)
+	testCreateStack(t, ctx)
+	testUpdateStack(t, ctx)
+	testDeleteStack(t, ctx)
```

Comment:
Let's use the same format used in the CLI where you have **t.Run**. See [dbusers_test.go#L51](https://github.com/mongodb/mongodb-atlas-cli/blob/7296e344b8ccad4032f62e24a5ee9aa70f9e3806/test/e2e/atlas/dbusers_test.go#L51)